### ISSUE TITLE
Grade bands are defined multiply and inconsistently

### ISSUE THREAD

**Issue by musoke** [OpenBeta/sandbag#91]
sandbag has at least 4 inconsistent sets of grade bands:

[src/GradeBands.ts](https://github.com/OpenBeta/sandbag/blob/b6a606bbd532066842bdc8e9aece3f1bdea41d83/src/GradeBands.ts#L5 )
```
export const GradeBands = {
  UNKNOWN: 'unknown',
  BEGINNER: 'beginner',
  INTERMEDIATE: 'intermediate',
  ADVANCED: 'advanced',
  EXPERT: 'expert'
} as const
```

[src/scales/index.ts](https://github.com/OpenBeta/sandbag/blob/b6a606bbd532066842bdc8e9aece3f1bdea41d83/src/scales/index.ts#L14) for routes
```
  band: 'beginner' | 'intermediate' | 'elite' | 'advanced'
```

[src/scales/index.ts](https://github.com/OpenBeta/sandbag/blob/b6a606bbd532066842bdc8e9aece3f1bdea41d83/src/scales/index.ts#L23) for boulders
```
  band: 'beginner' | 'intermediate' | 'elite' | 'experienced'
```

[src/data/routes.csv](https://github.com/OpenBeta/sandbag/blob/b6a606bbd532066842bdc8e9aece3f1bdea41d83/src/data/routes.csv) and 
[src/data/boulder.csv](https://github.com/OpenBeta/sandbag/blob/b6a606bbd532066842bdc8e9aece3f1bdea41d83/src/data/boulder.csv)
```
beginner, intermediate, experienced, expert, elite
```

It seems like the first set is what is [used in openbeta-graphql](), but I haven't checked carefully.  If that's the case, the last three cases can probably be dropped with minimal difficulty?

Looking further afield, they are also [reimplemented in open-tacos](https://github.com/OpenBeta/open-tacos/blob/develop/src/js/grades/bandUtil.ts) - I wonder if that's the right place for them since they are largely a UI matter?

**Comment by musoke** [comment]
@l4u532, hopefully this saves you some digging

**Comment by actuallyyun** [comment]
Is anyone working on this at the moment? If not, I would like to give it a go.

**Comment by l4u532** [comment]
Not sure when I’ll find the time. Feel free to dig in. Happy to give comments.

**Comment by musoke** [comment]
Ok!

**Comment by actuallyyun** [comment]
To confirm, the first set is indeed used in openbeta/GrahQL

```
export const CountByGradeBandSchema = new Schema<CountByGradeBandType>({
  unknown: { type: Number, required: true },
  beginner: { type: Number, required: true },
  intermediate: { type: Number, required: true },
  advanced: { type: Number, required: true },
  expert: { type: Number, required: true }
}, {
  _id: false
})
```

and also
`import { getScale, GradeBands, GradeBandTypes, GradeScalesTypes, isVScale } from '@openbeta/sandbag'`

I think we should drop the last three and use `sandbag` in opentacos

**Comment by musoke** [comment]
I thought the plan was to entirely remove grade bands and leave their definition to openbeta-graphql or open-tacos.  However, you're right that the bands defined in sandbag are used at https://github.com/OpenBeta/openbeta-graphql/blob/527798588d996f59388064538202a60d919886f2/src/db/utils/Aggregate.ts#L76, so we shouldn't do that.

**Comment by musoke** [comment]
fixed in #114
